README.rst

```diff
@@ -44,11 +44,476 @@ Available Features
+
+GET
++++
+
+Get details of an ``IP address`` instance.
+
+.. code-block:: text
+
+    /api/v1/ip-address/<ip_address-id>
+
+DELETE
+++++++
+
+Delete an ``IP address`` instance.
+
+.. code-block:: text
+
+    /api/v1/ip-address/<ip_address-id>
+
+PUT
++++
+
+Update details of an ``IP address`` instance.
+
+.. code-block:: text
+
+    /api/v1/ip-address/<ip_address-id>
+
+===========    ========================================
+Param          Description
+===========    ========================================
+ip_address     IPv6/IPv4 value
+subnet         Subnet UUID
+description    Optional description for the IP address
+===========    ========================================
+
+Export Subnet View
+##################
+
+View to export subnet data.
+
+POST
+++++
+
+.. code-block:: text
+
+    /api/v1/subnet/<subnet-id>/export
+
+Import Subnet View
+##################
+
+View to import subnet data.
+
+POST
+++++
+
+.. code-block:: text
+
+    /api/v1/import-subnet
+
+------------
+
+Exporting and Importing Subnet
+==============================
+
+One can easily import and export `Subnet` data and it's Ip Addresses using `openwisp-ipam`.
+This works for both IPv4 and IPv6 types of networks.
+
+Exporting
+#########
+
+Data can be exported via the admin interface or by using a management command. The exported data is in `.csv` file format.
+
+From management command
++++++++++++++++++++++++
+
+.. code-block:: shell
+
+    ./manage.py export_subnet <subnet value>
+
+This would export the subnet if it exists on the database.
+
+From admin interface
+++++++++++++++++++++
+
+Data can be exported from the admin interface by just clicking on the export button on the subnet's admin change view.
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-ipam/master/docs/images/export.png
+
+Importing
+#########
+
+Data can be imported via the admin interface or by using a management command.
+The imported data file can be in `.csv`, `.xls` and `.xlsx` format. While importing
+data for ip addresses, the system checks if the subnet specified in the import file exists or not.
+If the subnet does not exists it will be created while importing data.
+
+From management command
++++++++++++++++++++++++
+
+.. code-block:: shell
+
+    ./manage.py import_subnet --file=<file path>
+
+From admin interface
+++++++++++++++++++++
+
+Data can be imported from the admin interface by just clicking on the import button on the subnet view.
+
+.. image:: https://raw.githubusercontent.com/openwisp/openwisp-ipam/master/docs/images/import.png
+
+CSV file format
++++++++++++++++
+
+Follow the following structure while creating `csv` file to import data.
+
+.. code-block:: text
+
+    Subnet Name
+    Subnet Value
+
+    ip_address,description
+    <ip-address>,<optional-description>
+    <ip-address>,<optional-description>
+    <ip-address>,<optional-description>
+
+------------
+
+Extending openwisp-ipam
+=====================
+
+Extending API Views
+###################
```

Comment:
Don't we need instructions also on how to extend/reuse other parts?

- Models
- Admn
- AppConfig
- Tests

I think it should also be possible to provide a swapped model without having to create a different django app.
Would be nice to explain that as well and provide a working example.
We can reuse the same template for the other projects.